core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -740,6 +740,34 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_TRY_REMOVE_COUNT =
+      new Builder(Name.WORKER_BLOCK_REMOVER_TRY_REMOVE_COUNT)
+          .setDescription("The total number of blocks tried to be removed from this worker "
+              + "by asynchronous block remover.")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_REMOVED_COUNT =
+      new Builder(Name.WORKER_BLOCK_REMOVER_REMOVED_COUNT)
+          .setDescription("The total number of blocks was removed from this worker "
+              + "by asynchronous block remover.")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVE_SIZE =
```

Comment:
Will it be better to change to `WORKER_BLOCK_REMOVER_TRY_REMOVE_BLOCKS_SIZE`

Reply:
Sure